packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,146 @@
+
+2. Select the type of **Rate limit per client** that you want to implement:
+
+    * Select **Log and enforce client (recommended)** to enable client-based rate limiting.
+
+    * Select **Log per client** to enable client-based rate limiting in preview mode.
+
+    * Select **No action** to disable client-based rate limiting.
+
+### Frequently asked questions
+
+**Question:** Which endpoints are covered under client-based rate limiting?<br>
+**Answer:** Today, client-based rate limiting only applies to an authorization server's `/authorize` endpoint.
+
+**Question:** How is the client-specific rate limit determined?<br>
+**Answer:** The client rate-limiting framework calculates the per client rate limit based on the OAuth client ID, the user's IP address, and the Okta device identifier (the Okta device identifier that Okta sets in the browser).
+
+**Question:** What happens if my network contains a proxy server through which the requests are proxied?<br>
+**Answer:** Requests would appear to come from the same IP Address. When `/authorize` requests are made from behind a proxy IP address, make sure to [configure the respective IPs as proxies](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_Network). This allows the client-based rate-limiting framework to look for the IP address before the proxy to find the true client IP address.
+
+**Question:** Can I update the per client rate limiting today?<br>
+**Answer:** No. Today every client ID/IP address/device identifier combination gets 60 total requests per minute and a maximum of two concurrent requests.
+
+**Question:** Does the org-wide rate limit still apply when I enable client-based rate limiting?<br>
+**Answer:** Yes. The underlying org-wide rate limiting still applies to your Okta tenant. When the cumulative total request or maximum concurrent requests from every unique client exceeds the org-wide rate limits, your Okta tenant experiences org-wide rate-limit errors.
+
+**Question:** Would the rate-limit headers returned by Okta on the `/authorize` endpoint reflect client-specific rate limits?<br>
+**Answer:** Yes. The header value is specific to a given client/IP/device combination rather than the org-wide rate-limit values.
+
+**Question:** How can I find out if client-based rate limiting would be effective for my Okta tenant?<br>
+**Answer:** You can set the client-based rate limiting framework to **Log per client** mode. In **Log per client** mode, rate limiting is still based on the org-wide rate-limit values, but client-specific rate limiting error information is logged by firing System Log events. By analyzing these System Log events, you can determine if client-based rate limiting is effective for you.
+
+**Question:** Where does Okta get the device identifier from?<br>
+**Answer:** The device is identified using the `dt` cookie that Okta sets in the browser when the first request is made to Okta from the browser.
```

Comment:
This was already answered in the text above, not sure we need a separate question for it.